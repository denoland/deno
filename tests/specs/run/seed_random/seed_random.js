for (let i = 0; i < 10; ++i) {
  console.log(Math.random());
}

const arr = new Uint8Array(32);

crypto.getRandomValues(arr);
console.log(arr);

crypto.getRandomValues(arr);
console.log(arr);
