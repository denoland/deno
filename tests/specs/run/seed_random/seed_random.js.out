0.858562739044346
0.8973397944553141
0.15335012655691727
0.36867387434349963
0.3591039342838782
0.7044499748617652
0.7461423057751548
0.3824611207183364
0.5950178237266042
0.22440633214343908
Uint8Array(32) [
  153, 221, 127, 193, 173,  88,  77, 155,
   23,  66, 117, 239, 157, 231, 189, 160,
   79, 198,  30,  56, 137, 159, 220, 226,
   47, 211,  26,  73, 243, 252,  71, 214
]
Uint8Array(32) [
   18, 98,  66, 131,  76,  87,  93,  76,
  205, 81, 250, 112, 129, 119,  92,   9,
  116, 99,   5, 171,   8, 137, 132,  79,
  255,  9, 194,   1, 138,  85,  72, 189
]
